main.py

```
@@ -15,6 +15,7 @@
 import json
 import matplotlib.pyplot as plt
 import matplotlib
+from datetime import datetime
```

Ruff (0.11.9)

Redefinition of unused `datetime` from line 13

Remove definition: `datetime`

(F811)